### ISSUE TITLE
accessibilityIdentifier doesn't work on ASCellNode

### ISSUE THREAD

**Issue by wendyliga** [TextureGroup/Texture#1565]
the title explains itself

unable to set `accessibilityIdentifier` on ASCellNode inside `ASTableDataSource ASCellNodeBlock`

is this possible in the first place?

**Comment by maicki** [comment]
@wendyliga Could you please post some code how you are currently set the `accessibilityIdentifier`? The `accessibilityIdentifier` should be exposed and we are using it too without problems.

**Comment by wendyliga** [comment]
```
this is how I implements it

...
public func tableNode(_: ASTableNode, nodeBlockForRowAt indexPath: IndexPath) -> ASCellNodeBlock {
        guard data.count > indexPath.row else { return { ASCellNode() } }

        let item = data[indexPath.row]

        let cell = { () -> ASCellNode in
            let cell = CustomCell(data: item)
            cell.accessoryType = .disclosureIndicator
           cell.accessibilityIdentifier = "CustomCell"

            return cell
        }

        return cell
    }
...
```

**Comment by maicki** [comment]
Is `CustomCell` inheriting from `ASCellNode`? After rendering the view and look into Xcode's view debugger what is the `accessibilityIdentifier` set to of this cell?

**Comment by wendyliga** [comment]
@maicki yes, the `CustomCell` is inheriting `ASCellNode`, 

the `accessibilityIdentifier` after the rendering is 'nil'

**Comment by maicki** [comment]
@wendyliga Would it be possible to create a small example app that show's the issue, so we can look into it. Thank you!

**Comment by wendyliga** [comment]
@maicki  here you go https://github.com/wendyliga/cell-node

sorry take so long to create this example project

**Comment by maicki** [comment]
@wendyliga Hi, I just cloned your example and ran it and it seems for me the accessibilityIdentifier is set properly on the node and the view:

<img width="901" alt="Screen Shot 2019-07-04 at 9 16 08 AM" src="https://user-images.githubusercontent.com/25288/60679442-73d3d680-9e3c-11e9-8a88-e1b70e0780c1.png">

Maybe it could be the problem with your simulator that you have to run the Xcode Accessibility Inspector at least once so the accessibility support is provided within the Simulator going forward.

**Comment by wendyliga** [comment]
@maicki try your suggestion to open the `XCode Accessibility Inspector`, but the result is the same, even with `XCode Accessibility Inspector`, the value of the identifier is still `nil`

<img width="1680" alt="Screen Shot 2019-07-05 at 10 50 12" src="https://user-images.githubusercontent.com/16457495/60697081-bc5dc380-9f12-11e9-9445-02e416a6f5b0.png">

i also tried using real device, and still get `nil`

**Comment by maicki** [comment]
@wendyliga You are selecting the text node and not the cell node. You have to select the view behind the one you are currently selecting or set the identifier to the ASTextNode you have in that cell.

**Comment by wendyliga** [comment]
@maicki haha what a silly mistake, my bad man

thanks you
